### PR TITLE
OT208-79

### PR DESCRIPTION
**Pull request by MartinIriarte89**
1. Endpoint is added to list all the comments of a post.
2. The necessary method is added to list in service and repository.
3. Error codes are added in the exceptions thrown in the controller and path is protected in the security configuration.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controllers/CommentController.java [hunk 1]
```diff
@@ -106,7 +104,7 @@ public ResponseEntity<BasicCommentDto> editComment(
 		return ResponseEntity.ok(ObjectMapperUtils.map(comment, BasicCommentDto.class));
```
Comment (Oriana10) [inline src/main/java/com/alkemy/ong/controllers/CommentController.java:104]: Just as a suggestion, what if we make logic in the service and make controller as clean as it's possible?

Comment (MartinIriarte89) [reply src/main/java/com/alkemy/ong/controllers/CommentController.java]: corrected

File: src/main/java/com/alkemy/ong/services/CommentService.java [hunk 2]
```diff
@@ -8,4 +9,12 @@ public interface CommentService extends BasicService<CommentEntity, String> {
 
 	List<CommentEntity> findAllOrderByTimestamps();
 
+	List<CommentEntity> findAllByNewsId(String id);
+
+	<D, T> D map(final T entity, Class<D> outClass);
```
Comment (acamus79) [inline src/main/java/com/alkemy/ong/services/CommentService.java:14]: Me parece que eso ya lo tenes en el ObjetMapperUtils, volverlo hacer aca es un desproposito

**Review by acamus79 — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/services/CommentService.java (hunk 2 above)
Comment (MartinIriarte89) [reply src/main/java/com/alkemy/ong/services/CommentService.java]: Corrected

**Review by EdwardDavys — APPROVED** [review]
Approve

**Review by acamus79 — APPROVED** [review]

**Pull request merged**